schema/spacewalk/upgrade/susemanager-schema-4.4.4-to-susemanager-schema-4.4.5/011-update-queue_image.sql

```diff
@@ -0,0 +1,42 @@
+--
+-- Copyright (c) 2017 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+--
+
+CREATE OR REPLACE FUNCTION
+queue_image(image_id_in IN NUMERIC, immediate_in IN NUMERIC DEFAULT 1)
```

Comment:
I don't see that function used anywhere... Is that expected? It's also only in the upgrade scripts.

Reply:
It is used: in rhn_channel.update_needed_cache()

Reply:
Or right... those are not new, just changed. My bad.